lib/screens/SecondScreen.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:module_nine/model/photo.dart';
+import 'package:module_nine/services/app_routes.dart';
+import 'package:module_nine/services/network_service.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class SecondScreen extends StatelessWidget {
+  const SecondScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Directionality(
+      textDirection: TextDirection.ltr,
+      child: Scaffold(
+        appBar: AppBar(
+          centerTitle: true,
+          title: Text(AppLocalizations.of(context)!.secondScreen),
+        ),
+        body: FutureBuilder(
+          future: NetworkService().get(),
+          builder: (context, snapshot) {
+            if (!snapshot.hasData) {
+              return const Center(
+                child: CircularProgressIndicator(),
+              );
+            } else if (snapshot.hasError) {
+              return const Center(
+                child: Text('Error'),
+              );
+            }
+            return ListView(
+              children: (snapshot.data as List<Photo>)
+                  .map(
+                    (photo) => ListTile(
```

Comment:
I recommend extracting Tile in a dedicated Widget.

Reply:
done